spec/diagnose_spec.rb

```diff
@@ -1187,28 +1287,51 @@
   end
 
   it "prints agent diagnose section with errors" do
+    expected_output = if @runner.type == :python
+                        [
+                          /Agent diagnostics/,
+                          /  Extension tests/,
```

Comment:
Since the Python integration has no extension, let's omit this to avoid confusion.

We can make the 'Configuration test' line below it part of the Agent tests section. It's from the agent diagnose report anyway.